packages/calcite-components/src/components/dropdown-item/dropdown-item.scss

```diff
@@ -1,11 +1,25 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-dropdown-item-background-color-hover: Specifies the background color of the dropdown item when hovered.
+ * @prop --calcite-dropdown-item-background-color-press: Specifies the background color of the dropdown item when selected or active.
+ * @prop --calcite-dropdown-item-icon-color-hover: Specifies the color of the dropdown item icon when hovered. 
```

Comment:
Nitpick: It does seem like having a "base" for `text-color` but not `icon-color` here is a bit unexpected if there is a hover state for `icon-color`. These would be the `icon-start/end` right? 

There could be a need to have those be distinct and independent from other overrides like the selection icon color, but we can add as requested!

Reply:
I agree but I think we should wait for our customers to tell us.